### PR TITLE
docs: bench setup requirements --dev

### PR DESCRIPTION
**Pull request by ankush**
Docs for dev requirement setup feature.

https://github.com/frappe/bench/pull/1196

### CODE REVIEW

File: frappe_docs/www/docs/user/en/basics/apps.md [hunk 1]
```diff
@@ -81,11 +82,19 @@ apps/custom_app
 This file stores the list of python dependencies. When this app is installed,
 it's dependencies will also get installed.
 
+
```
Comment (ankush) [inline frappe_docs/www/docs/user/en/basics/apps.md:85]:
```suggestion
```

unrelated

**Review by gavindsouza — APPROVED** [review]
